RELEASE_NOTES.md

```diff
@@ -19,17 +19,115 @@
+* IOTDB-161	Add ErrorCode of different response errors
+* IOTDB-160	External sort
+* IOTDB-153	Further limit fetchSize to speed up LIMIT&OFFSET query
+* IOTDB-295	Refactor db.exception
+* IOTDB-257 Makes the client stop fetch when dataSize equals maxPrintRowCount and change client fetchSize less than maxPrintRowCount
+* IOTDB-215 Update TsFile sketch tool and TsFile docs for v0.9.0
+* IOTDB-285 Duplicate fields in EngineDataSetWithoutValueFilter.java
+* IOTDB-210 One else if branch will never be reached in the method optimize of ExpressionOptimizer
+* IOTDB-276 Fix inconsistent ways of judging whether a Field is null
+* IOTDB-273 Parallel recovery
+* IOTDB-287 Restrict users to only use domain names and IP addresses.
+* IOTDB-293 Variable naming convention
+* IOTDB-295 Refactor db.exception
+* IOTDB-144 meta data cache for query
+* Reconstruct Antlr3 grammar to improve performance
+* Tooling for release
+* Modified Decoder and SequenceReader to support old version of TsFile 
+* Remove jdk constrain of jdk8 and 11
+* Modify print function in AbstractClient
+* Avoid second execution of parseSQLToPhysicalPlan in executeStatement
+
+## Known Issues
+
+* IOTDB-20  Need to support UPDATE
+
+## Bug Fixes
 
-* New Features
-* Incompatible changes
-* Miscellaneous changes
-* Known Issues
-* Bug Fixes
+* IOTDB-266 NullPoint exception when reading not existed devices using ReadOnlyTsFile
+* IOTDB-264	Restart failure due to WAL replay error
+* IOTDB-240	Fix unknown time series in where clause
+* IOTDB-168&169 Fix a bug in export-csv tool and fix compatibility of timestamp formats in exportCsv, client display and sql
+* IOTDB-174	Fix querying timeseries interface cannot make a query by the specified path prefix
+* IOTDB-195	Using String.getBytes(utf-9).length to replace string.length() in ChunkGroupMetadata for supporting Chinese
+* IOTDB-211	Use "%IOTDB_HOME%\lib\*" to refers to all .jar files in the directory in start-server.bat
+* IOTDB-244 Fix bug when querying with duplicated columns
+* IOTDB-252 Add/fix shell and bat for TsFileSketchTool/TsFileResourcePrinter
+* IOTDB-290 Bug about threadlocal field in TSServiceImpl.java
+* IOTDB-291 Statement close operation may cause the whole connection's resource to be released
+* IOTDB-296 Fix error when skip page data in sequence reader
+* Fix Dynamic Config when Creating Existing SG or Time-series
+* Fix start-walchecker scripts for leting user define the wal folder
+
+
+# Apache IoTDB (incubating) 0.8.1
+
+This is a bug-fix version of 0.8.0
+
+- [IOTDB-172] bug in updating startTime and endTime in TsFileResource
+- Abnormal publishing of sequence and unsequence data folders in DirectoryManager
+- fix a bug in TimeRange's intersects function
+- [IOTDB-202] fix tsfile example data type
+- [IOTDB-195] Bug about 'serializedSize' in ChunkGroupMetaData.java (for Chinese string)
+- [IOTDB-242] fix mvn integration-test failed because the files in the target folder changes
```

Comment:
Forgive me for nitpicking, but please keep the format consistent as you can.
I see that some are surrounded with [] such as [IOTDB-202] and some are not. 
Some are indented with short spaces and some are with longer spaces. For example:
![image](https://user-images.githubusercontent.com/33376433/68563717-6cc37180-0489-11ea-80be-9917979bf6de.png)

Reply:
Also, sorting the item by their IOTDB-XXX number is a good habit.

Reply:
Thanks, I do not notice it when using the idea